Lib/idlelib/calltips.py

```diff
@@ -123,6 +123,8 @@ def get_entity(expression):
 _INDENT = ' '*4  # for wrapped signatures
 _first_param = re.compile(r'(?<=\()\w*\,?\s*')
 _default_callable_argspec = "See source or doc"
+_invalid_method = "invalid method signature"
+_argument_positional = "('/' marks preceding arguments as positional-only)"
```

Comment:
I'll prefer `['/' marks preceding arguments as positional-only]` style, it has more conspicuous when the text was end at the box margin.

Also, I think the word is OK, using google to search will got this [stackoverflow post](https://stackoverflow.com/questions/24735311/python-what-does-the-slash-mean-in-the-output-of-helprange) at the top, it should not be a problem for newbie.